build/docker/alpine.Dockerfile

```
@@ -54,6 +54,9 @@ RUN apk --no-cache --update add \
 
 RUN dotnet --version && npm -v && yarn -v
 
+RUN npm install --global bower
+RUN bower -v
```

Do these in one line:

```suggestion
RUN npm install --global bower && bower -v
```